helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1331,3 +1331,15 @@ networkExplorer:
       # seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
       # seccomp.security.alpha.kubernetes.io/defaultProfileName: 'docker/default'
       # apparmor.security.beta.kubernetes.io/defaultProfileName: 'runtime/default'
+
+################################################################################
+# Open-telemetry operator running as a deployment with a replica count of 1.
+# It auto-instruments applications to send telemetry data to the Splunk backend.
```

Comment:
To avoid sounding like applications will be sending data directly to the backend. We should sent to local agents instead, right?

```suggestion
# It auto-instruments applications to emit telemetry data.
```